site/docs/compositemark.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: docs
+menu: docs
+title: Composite Mark
+permalink: /docs/compositemark.html
+---
+
+Composite marks are "macros" for more complex layered mark specs. Layered mark specs create visualizations with a series of basic mark types.
```

Comment:
Composite marks are "macros" for creating more complex layered graphics with multiple primitive marks.  Currently, we include only one composite mark type: `box-plot`.